README.md

```diff
@@ -108,6 +108,20 @@ These are the fields that are turned into TemplateFields when the YAML files are
 *Note:*
 *All markdown formatting, HTML tags, or jinja passed in as part of the content of variables will be escaped.*
 
+Development
+-----------
+
+A local checkout of the frameworks repo can be shared with locally-running services (i.e. frontend applications)
+as follows:
+
+- from this repo, run `bower link`
+- from each app, run `bower link digitalmarketplace-frameworks`
+
+Then, whenever the framework content is changed:
+
+- deploy framework content to each frontend app with `make frontend_build` (NB this step required before deployment to PaaS via `cf push`)
```

Comment:
I think "rebuild" would probably fit better than "deploy". Also I'm not sure if it's clear that you need to run the make step in the app itself, not in the frameworks repo.